src/semaphore/config.py

```diff
@@ -141,7 +147,7 @@ def validate_github_app(cls, v: bool, values: Mapping[str, Any]) -> bool:
     @property
     def arq_redis_settings(self) -> RedisSettings:
         """Create a Redis settings instance for arq."""
-        url_parts = urlparse(self.redis_queue_url)
+        url_parts = urlparse(self.redis_url)
```

Comment:
We'll use a separate Redis URL just for the arq queue, so add a new redis field to the configs.